lesson6/main.js

```diff
@@ -0,0 +1,33 @@
+const ul = document.getElementById('js-ul');
+const frag = document.createDocumentFragment();
```

Comment:
Since it's only called once, there seems to be no need to declare it globally.

IMO, I would declare it in "then".

Reply:
Thanks for the review!
Fixed.
I learned that declaring variables in the global scope leads to memory consumption.
83612a9fee2a32a9e8842a575fc569d10d470446